Formula/polynote.rb

```diff
@@ -0,0 +1,25 @@
+class Polynote < Formula
+  desc "The polyglot notebook with first-class Scala support"
+  homepage "https://polynote.org/"
+  url "https://github.com/polynote/polynote/releases/download/0.2.8/polynote-dist.tar.gz"
+  version "0.2.8"
+  sha256 "97d15fb866269885d79218e7ed97fbe188b88d0afbf4933d7d2a6b452a857c29"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install Dir["*"]
+
+    (bin/"polynote").write <<~EOS
+      #!/usr/bin/env bash
+      (cd #{libexec} ; ./polynote)
+    EOS
+  end
+
+  test do
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.